packages/atlas/src/components/_comments/CommentInput/CommentInput.tsx

```diff
@@ -27,11 +27,12 @@ export type CommentInputProps = {
   processing: boolean
   readOnly?: boolean
   memberHandle?: string
+  value: string
+  hasInitialValueChanged: boolean
   onComment?: () => void
   onCancel?: () => void
-  onChange?: (e: React.ChangeEvent<HTMLTextAreaElement>) => void
+  onChange?: (value: string) => void
```

Comment:
Now we don't need that change. I would bring back `onChange` which takes the event as a parameter.

Reply:
fixed